incubator/command-config/src/main/java/io/aklivity/zilla/runtime/command/config/internal/openapi/http/proxy/OpenApiHttpProxyConfigGenerator.java

```diff
@@ -0,0 +1,519 @@
+                        .build()
+                    .exit(tlsExit)
+                    .build();
+        }
+        else if (isPlainEnabled)
+        {
+            binding
+                .route()
+                    .exit(plainExit)
+                    .build();
+        }
+        else if (isTlsEnabled)
+        {
+            binding
+                .route()
+                    .exit(tlsExit)
+                    .build();
+        }
+        return binding;
+    }
+
+    private HttpOptionsConfigBuilder<BindingConfigBuilder<NamespaceConfigBuilder<NamespaceConfig>>> injectHttpServerOptions(
+        HttpOptionsConfigBuilder<BindingConfigBuilder<NamespaceConfigBuilder<NamespaceConfig>>> options)
+    {
+        if (isJwtEnabled)
+        {
+            options
+                .authorization()
+                    .name("jwt0")
+                        .credentials()
+                            .header()
+                                .name("authorization")
+                                .pattern("Bearer {credentials}")
+                                .build()
+                        .build()
+                    .build();
+        }
+        return options;
+    }
+
+    private BindingConfigBuilder<NamespaceConfigBuilder<NamespaceConfig>> injectHttpServerRoutes(
+        BindingConfigBuilder<NamespaceConfigBuilder<NamespaceConfig>> binding,
+        String exit)
+    {
+        for (String path : openApi.paths.keySet())
+        {
+            PathItem2 item = PathItem2.of(openApi.paths.get(path));
+            for (String method : item.methods().keySet())
+            {
+                binding
+                    .route()
+                    .exit(exit)
+                    .when(HttpConditionConfig::builder)
+                        .header(":path", path.replaceAll("\\{[^}]+\\}", "*"))
+                        .header(":method", method)
+                        .build()
+                    .inject(route -> injectHttpServerRouteGuarded(route, item, method))
+                    .build();
+            }
+        }
+        return binding;
+    }
+
+    private RouteConfigBuilder<BindingConfigBuilder<NamespaceConfigBuilder<NamespaceConfig>>> injectHttpServerRouteGuarded(
+        RouteConfigBuilder<BindingConfigBuilder<NamespaceConfigBuilder<NamespaceConfig>>> route,
+        PathItem2 item,
+        String method)
+    {
+        List<Map<String, List<String>>> security = item.methods().get(method).security;
+        if (security != null)
+        {
+            for (Map<String, List<String>> securityItem : security)
+            {
+                for (String securityItemLabel : securityItem.keySet())
+                {
+                    if (isJwtEnabled && "jwt".equals(securitySchemes.get(securityItemLabel)))
+                    {
+                        route
+                            .guarded()
+                            .name("jwt0")
+                            .role("echo:stream")
```

Comment:
Perhaps this should be omitted since it doesn't have meaning in the context of the OpenAPI source?

Reply:
Removing the role results in this output:
```
    routes:
    - exit: http_client0
      when:
      - headers:
          :path: /items
          :method: GET
      guarded:
        jwt0: []
```
Pls confirm if this is OK.

Reply:
Yes, this means they must be a valid user, but no specific roles are required.

Looks like this is the official syntax we are looking for to drive the generation, agree?
https://github.com/OAI/OpenAPI-Specification/pull/2515/files